src/services/notificationService.ts

```diff
@@ -30,7 +30,7 @@ import {
  */
 const APNSTemplate: INotificationTemplate = {
   body:
-    '{"aps": {"alert": {"title": "$(title)", "body": "$(message)"}}, "message_id": "$(message_id)"}'
+    '{"aps": {"alert": {"title": "$(title)", "body": "$(message)"}}, "message_id": "$(message_id)", "deep_link": "italiaapp://$(deep_link)"}'
```

Comment:
I would remove the `body` since it's almost always going to be too long to see here (and formatted with markdown).